tests/trigger.spec.ts

```diff
@@ -233,7 +233,8 @@ describe('trigger', () => {
       const wrapper = mount(Component, {})
 
       for (const keyName in keyCodesByKeyName) {
-        const keyCode = keyCodesByKeyName[keyName]
+        const keyCode =
+          keyCodesByKeyName[keyName as keyof typeof keyCodesByKeyName]
```

Comment:
hard to read 😅 ain't that just `number`?

Reply:
not quite, it's a string, union of keys of the `keyCodesByKeyName` to be exact
Now I imported the `KeyName` type instead.